worlds/shorthike/__init__.py

```diff
@@ -41,11 +41,8 @@ class ShortHikeWorld(World):
 
     required_client_version = (0, 4, 4)
 
-    def __init__(self, multiworld, player):
-        super(ShortHikeWorld, self).__init__(multiworld, player)
-
     def get_filler_item_name(self) -> str:
-        return "13 Coins"
+        return self.options.filler_coin_amount.current_option_name
```

Comment:
```suggestion
        return self.options.filler_coin_amount.current_key
```

Reply:
actually is this because it needs to be capitalized? if so it's probably right, just looks weird

Reply:
Yeah `current_option_name` returns the correct value necessary for the item.